Detectors/TPC/workflow/src/CATrackerSpec.cxx

```diff
@@ -623,6 +624,28 @@ DataProcessorSpec getCATrackerSpec(ca::Config const& specconfig, std::vector<int
         ClusterNativeHelper::Reader::fillIndex(clusterIndex, clusterBuffer, clustersMCBuffer, inputs, mcInputs, [&validInputs](auto& index) { return validInputs.test(index); });
         ptrs.clusters = &clusterIndex;
       }
+      // a byte size resizable vector object, the DataAllocator returns reference to internal object
+      // initialize optional pointer to the vector object
+      using ClusterOutputChunkType = std::decay_t<decltype(pc.outputs().make<std::vector<char>>(Output{"", "", 0}))>;
+      ClusterOutputChunkType* clusterOutput = nullptr;
```

Comment:
@davidrohr , providing the buffer for cluster output now before calling the GPU worker, so you can use it later to directly write to the vector. It has zero length now, although created there.